zargcon.md

```diff
@@ -0,0 +1,47 @@
+---
+title: ZargCon Rules
+layout: standard-page
+---
+
+# ZargCon
+
+ZargCon is an annual weekend of gaming: board, video, roleplaying, and otherwise. It began with a small group in 2015.
+
+## How much do I need to pay?
+
+$5 USD per person in your group. ZargCon is a volunteer effort and our fees are always intended to cover our bare minimum expenses.
+
+## Event Rules
+
+1. No children under 13 years of age may attend
+1. Everyone, regardless of age, should act in a mature, polite, and empathetic manner at all times
+    * Failure to do so will result in a warning, followed by being asked to leave the event for its duration
+1. Label any homemade dishes with allergen information
+1. All attendees are expected to adhere to the Zargon Organization bylaws (https://zrgn.github.io/)
+
+## When is it?
+
+7—8 December 2019
+
+## Where is it?
+
+In 2019, we are being hosted by Red Hat Tower Annex.
+
+* Because this is the place of employment for many of our members, we ask that all attendees take extra care to respect the venue
+* Drinks and snacks provided by Red Hat will not be available to our attendees
+
+## What should I bring?
+
+* Yourself
+* Any games you want to play
+* Snacks & Beverages
+
+## FAQ
+
+### Can I invite other people?
+
+As long as you're cool with them and you think they won't be jerks, sure! They'll still have to pay however. We will also need a list names for any of your guests.
+
+### Is there a registration cut off?
+
+Yes. Monday Dec 2. Have your registration fee paid and provide your guest list by then.
```

Comment:
We also need to tell people how exactly they can register.

Reply:
Edited.